### ISSUE TITLE
BW 6.7 Plug-in issue - [ERROR] error reading ~/.m2/repository/com/tibco/plugins/com.tibco.xml.cxf.common/1.3.200/com.tibco.xml.cxf.common-1.3.200.jar; error in opening zip file

### ISSUE THREAD

**Issue by carlosaploura** [TIBCOSoftware/bw6-plugin-maven#655]
Maven Plugin version: 2.8.1

Maven version:

     Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
     Maven home: C:\Software\apache-maven-3.6.3\bin\..
     Java version: 1.8.0_221, vendor: Oracle Corporation, runtime: C:\Tibco670\tibcojre64\1.8.0
     Default locale: en_US, platform encoding: Cp1252
     OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"

Product : TIBCO ActiveMatrix Businessworks 6

Product version: 6.7.0 HF001

Component: Maven build plugin

**Steps to reproduce the issue:**
1. Install Maven and BW6 Maven plug-in v2.8.1
2. Generate POM files
3. Execute command "mvn package"

**Describe the results you received:**

The JAR file "com.tibco.xml.cxf.common-1.3.200.jar" is downloaded but it's file size is only 1KB.

Because of this, I get the following error:

     [ERROR] COMPILATION ERROR :
     [INFO] -------------------------------------------------------------
     [ERROR] error reading ~\.m2\repository\com\tibco\plugins\com.tibco.xml.cxf.common\1.3.200\com.tibco.xml.cxf.common-1.3.200.jar; error in opening zip file
     [ERROR] error reading ~\.m2\repository\com\tibco\plugins\com.tibco.xml.cxf.common\1.3.200\com.tibco.xml.cxf.common-1.3.200.jar; error in opening zip file
     [ERROR] ~/TA_Poms/nl.apg.tibco.custom/src/nl/apg/tibco/custom/CustomFunctions.java:[3,44] package com.tibco.xml.cxf.common.annotations does not exist
     [ERROR] ~/TA_Poms/nl.apg.tibco.custom/src/nl/apg/tibco/custom/CustomFunctions.java:[4,44] package com.tibco.xml.cxf.common.annotations does not exist
     [ERROR] ~/TA_Poms/nl.apg.tibco.custom/src/nl/apg/tibco/custom/CustomFunctions.java:[5,44] package com.tibco.xml.cxf.common.annotations does not exist
     [ERROR] ~/TA_Poms/nl.apg.tibco.custom/src/nl/apg/tibco/custom/CustomFunctions.java:[11,26] package com.tibco.security does not exist
     [ERROR] ~/TA_Poms/nl.apg.tibco.custom/src/nl/apg/tibco/custom/CustomFunctions.java:[12,26] package com.tibco.security does not exist
     [ERROR] ~/TA_Poms/nl.apg.tibco.custom/src/nl/apg/tibco/custom/CustomFunctions.java:[15,2] cannot find symbol
       symbol: class XPathFunctionGroup
     [ERROR] ~/TA_Poms/nl.apg.tibco.custom/src/nl/apg/tibco/custom/CustomFunctions.java:[62,84] cannot find symbol
       symbol:   class AXSecurityException
       location: class nl.apg.tibco.custom.CustomFunctions
     [ERROR] ~/TA_Poms/nl.apg.tibco.custom/src/nl/apg/tibco/custom/CustomFunctions.java:[22,10] cannot find symbol
       symbol:   class XPathFunction
       location: class nl.apg.tibco.custom.CustomFunctions
     [ERROR] ~/TA_Poms/nl.apg.tibco.custom/src/nl/apg/tibco/custom/CustomFunctions.java:[60,10] cannot find symbol
       symbol:   class XPathFunction
       location: class nl.apg.tibco.custom.CustomFunctions
     [ERROR] ~/TA_Poms/nl.apg.tibco.custom/src/nl/apg/tibco/custom/CustomFunctions.java:[64,45] cannot find symbol
       symbol:   variable ObfuscationEngine
       location: class nl.apg.tibco.custom.CustomFunctions

**Describe the results you expected:**

I expected the "com.tibco.xml.cxf.common-1.3.200.jar" to be successfully downloaded and to generate an EAR file for my application.

**Additional information you deem important (e.g. issue happens only occasionally):**

The issue only happens when I have a dependency on a _Custom Functions_ module. Otherwise the build finishes successfully.

**Comment by sarundes-tibco** [comment]
@carlosaploura Can you please check on latest bw6-maven-plugin 2.9.9.

**Comment by ukuppa-tibco** [comment]
JIRA opened https://jira.tibco.com/browse/AMBW-51412 and upon investigation we discovered that the issue is linked to the Apache Maven version. We tested it up to Maven version 3.8.8, where it worked without issues. However, the problem is reproducible with version 3.9.9.
According to Dev team; we do claim support for our Maven plugin component with Apache Maven 3.8.x. Hence closing this ticket for now.
